src/panel/pages/events/Timeline.tsx

```diff
@@ -220,12 +228,15 @@ const TimelineIcons = styled.div`
 
 const TimelineList = styled.div`
   cursor: grab;
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
+  display: block;
   position: relative;
   padding: 70px 0;
-  overflow: hidden;
+  overflow-y: visible;
+  width: 100%;
+  box-sizing: border-box;
```

Comment:
Not that it really matters, but I think alphabeticalising(?) css props can help keep them at least somewhat organised

Reply:
Would be cool if prettier could do this - do you have examples of repos that use this? 

I can see the advantages - sometimes I group by type (e.g. color, positioning) but I'm not consistent with that.

Reply:
No concrete examples to hand - I don't think there's a defacto standard

I just found [this post](https://dev.to/thekashey/happy-potter-and-the-order-of-css-5ec) that was a fairly interesting read, and mentions [stylelint-config-rational-order](https://www.npmjs.com/package/stylelint-config-rational-order) and [stylelint-order](https://www.npmjs.com/package/stylelint-order) which could be useful if using `stylelint`